tests/ergocub-model-test.cpp

```diff
@@ -449,7 +449,7 @@ bool checkFTSensorsAreOddAndNotNull(iDynTree::ModelLoader & mdlLoader)
 }
 
 /**
- * All the ergoCub have a even and not null number of F/T sensors.
+ * All the iCub have a even and not null number of F/T sensors.
```

Comment:
Is this change intentional?

Reply:
No I fixed in the previous commit and reverting it put back iCub